src/ModelContextProtocol/Server/AIFunctionMcpServerTool.cs

```diff
@@ -195,57 +195,59 @@ public override async Task<CallToolResponse> InvokeAsync(
             };
         }
 
-        switch (result)
+        return result switch
         {
-            case null:
-                return new()
-                {
-                    Content = []
-                };
-
-            case string text:
-                return new()
-                {
-                    Content = [new() { Text = text, Type = "text" }]
-                };
-
-            case TextContent textContent:
-                return new()
-                {
-                    Content = [new() { Text = textContent.Text, Type = "text" }]
-                };
-
-            case DataContent dataContent:
-                return new()
-                {
-                    Content = [new()
+            null => new()
+            {
+                Content = []
+            },
+            string text => new()
+            {
+                Content = [new() { Text = text, Type = "text" }]
+            },
+            TextContent textContent => new()
+            {
+                Content = [new() { Text = textContent.Text, Type = "text" }]
+            },
+            DataContent dataContent => new()
+            {
+                Content = [new()
                     {
                         Data = dataContent.GetBase64Data(),
                         MimeType = dataContent.MediaType,
                         Type = dataContent.HasTopLevelMediaType("image") ? "image" : "resource",
```

Comment:
As long as you're here, the spec now includes explicit mention of audio... can you update this (and other similar cases) to be:
```C#
Type =
    dataContent.HasTopLevelMediaType("image") ? "image" :
    dataContent.HasTopLevelMediaType("audio") ? "audio" :
    "resource",
```